etc/README.md

```diff
@@ -1,8 +1,31 @@
 # Installation instructions
 
-## bash + Homebrew
+## Homebrew
 
-If you're using Homebrew, just run `brew install hub` and you should be all set with auto-completion.
+If you're using Homebrew, just run `brew install hub` and you should be all set with auto-completion (for both Bash and Zsh).
```

Comment:
This is not true (for zsh). The reader might think that they are done at this point. However, if they also installed git with Homebrew, then their hub (when aliased as "git") completions are busted.

This sentence should add that you're all set **for bash**, but that you should read on for zsh instructions.